apps/src/templates/Button.story.jsx

```diff
@@ -2,135 +2,108 @@ import React from 'react';
 import Button from './Button';
 import {action} from '@storybook/addon-actions';
 
-export default storybook => {
```

Comment:
storybook files were migrated to the new syntax; all the render behavior is unchanged.